compiler/src/iree/compiler/Codegen/LLVMGPU/test/ROCDL/pipeline_tile_and_fuse.mlir

```diff
@@ -718,3 +718,109 @@ hal.executable public @main {
+            affine_map<(d0, d1, d2) -> (d1, d2)>,
+            affine_map<(d0, d1, d2) -> (d0, d1)>],
+          iterator_types = [
+            #iree_gpu.iterator_type<parallel>,
+            #iree_gpu.iterator_type<parallel>,
+            #iree_gpu.iterator_type<reduction>],
+          kind = #iree_gpu.data_tiled_mma_layout<
+            intrinsic = MFMA_F32_16x16x4_F32,
+            unroll_m = 8,
+            unroll_n = 2,
+            unroll_n_to_subgroups = 4,
+            unroll_k = 4>}
+          : tensor<1x1x8x4x16x4xf32>, tensor<1x1x4x2x4x16x4xf32> into tensor<1x1x8x4x2x4x16x4xf32>
+        flow.dispatch.tensor.store %6, %2, offsets = [0, 0, 0, 0, 0, 0, 0, 0], sizes = [1, 1, 8, 4, 2, 4, 16, 4], strides = [1, 1, 1, 1, 1, 1, 1, 1] : tensor<1x1x8x4x2x4x16x4xf32> -> !flow.dispatch.tensor<readwrite:tensor<1x1x8x4x2x4x16x4xf32>>
+        return
+      }
+    }
+  }
+}
+
+// CHECK-LABEL: func.func @multi_mma_data_tiled_unrolled_MFMA_F32_16x16x4_F32()
+// CHECK-DAG:  %[[A_READ:.+]] = vector.transfer_read {{.+}}, vector<8x1x1x4xf32>
+// CHECK-DAG:  %[[B_READ:.+]] = vector.transfer_read {{.+}}, vector<2x1x1x4xf32>
+// CHECK-DAG:  %[[C_READ:.+]] = vector.transfer_read {{.+}}, vector<8x1x2x1x1x4xf32>
+// CHECK-DAG:  %[[C_SLICE00:.+]] = vector.extract_strided_slice %[[C_READ]] {{.*}}offsets = [0, 0, 0, 0, 0, 0]{{.*}} : vector<8x1x2x1x1x4xf32> to vector<1x1x1x1x1x4xf32>
+// CHECK-DAG:  %[[C_SLICE01:.+]] = vector.extract_strided_slice %[[C_READ]] {{.*}}offsets = [0, 0, 1, 0, 0, 0]{{.*}} : vector<8x1x2x1x1x4xf32> to vector<1x1x1x1x1x4xf32>
+// CHECK-DAG:  %[[C_SLICE70:.+]] = vector.extract_strided_slice %[[C_READ]] {{.*}}offsets = [7, 0, 0, 0, 0, 0]{{.*}} : vector<8x1x2x1x1x4xf32> to vector<1x1x1x1x1x4xf32>
+// CHECK-DAG:  %[[C_SLICE71:.+]] = vector.extract_strided_slice %[[C_READ]] {{.*}}offsets = [7, 0, 1, 0, 0, 0]{{.*}} : vector<8x1x2x1x1x4xf32> to vector<1x1x1x1x1x4xf32>
+// CHECK-DAG:  %[[C_00_0:.+]] = vector.shape_cast %[[C_SLICE00]] : vector<1x1x1x1x1x4xf32> to vector<4xf32>
+// CHECK-DAG:  %[[C_01_0:.+]] = vector.shape_cast %[[C_SLICE01]] : vector<1x1x1x1x1x4xf32> to vector<4xf32>
+// CHECK-DAG:  %[[C_70_0:.+]] = vector.shape_cast %[[C_SLICE70]] : vector<1x1x1x1x1x4xf32> to vector<4xf32>
+// CHECK-DAG:  %[[C_71_0:.+]] = vector.shape_cast %[[C_SLICE71]] : vector<1x1x1x1x1x4xf32> to vector<4xf32>
+// CHECK-DAG:  %[[A_EXTRACT00:.+]] = vector.extract %[[A_READ]][0, 0, 0, 0] : f32 from vector<8x1x1x4xf32>
+// CHECK-DAG:  %[[A_EXTRACT01:.+]] = vector.extract %[[A_READ]][0, 0, 0, 1] : f32 from vector<8x1x1x4xf32>
+// CHECK-DAG:  %[[A_EXTRACT02:.+]] = vector.extract %[[A_READ]][0, 0, 0, 2] : f32 from vector<8x1x1x4xf32>
+// CHECK-DAG:  %[[A_EXTRACT03:.+]] = vector.extract %[[A_READ]][0, 0, 0, 3] : f32 from vector<8x1x1x4xf32>
+// CHECK-DAG:  %[[A_EXTRACT70:.+]] = vector.extract %[[A_READ]][7, 0, 0, 0] : f32 from vector<8x1x1x4xf32>
+// CHECK-DAG:  %[[A_EXTRACT71:.+]] = vector.extract %[[A_READ]][7, 0, 0, 1] : f32 from vector<8x1x1x4xf32>
```

Comment:
You can add `CHECK-COUNT-24` between these to make sure there are the right number of slices without adding too many lines here, i.e.:
```
// CHECK-COUNT-24: vector.extract %[[A_READ]][{{[0-9]+}}, 0, 0, 1] : f32 from vector<8x1x1x4xf32>
```

Reply:
The problem is that these different kinds of lines are interleaved, which is why CHECK-DAG helps here.